pandas/io/formats/style_render.py

```diff
@@ -1345,6 +1348,165 @@ def format_index(
 
         return self
 
+    def relabel_index(
```

Comment:
Hmm maybe `relabel_axis` since it can operate on column labels as well?

Reply:
The other Styler functions, `format_index`, `applymap_index` and `apply_index`, can all operate on column labels as well. There was a discussion originally about this but I can't remember the final reasoning for going with "index" rather than "axis". I think it was to highlight the difference between `apply` which can act along a different `axis` whilst `apply_index` applies to `Index` or `MultiIndex` either as row or column indexes.

Reply:
Gotcha. Aligning with those other names makes sense then